### PR TITLE
Config iter5

### PR DESCRIPTION
**Pull request by yuvrajzala26**

### CODE REVIEW

**Review by AashnaNarang — CHANGES_REQUESTED** [review]
You need to also add the timing for how the door should stay open

File: src/main/Elevator.java [hunk 1]
```diff
@@ -206,6 +217,12 @@ public void run() {
...
 		    };
+		    try {
+				Thread.sleep(Configurations.TIME_TO_LOAD_UNLOAD);
+			} catch (InterruptedException e) {
+				// TODO Auto-generated catch block
+				e.printStackTrace();
+			}
 		    Timer timer = new Timer("Timer");
 		    long delay = fe == null ? 500 : 
 		    	fe.getErrorCode() == 1 ? 1000 : 500;
```
Comment (AashnaNarang) [inline src/main/Elevator.java]: The delay is basically how long the door timer is, basically change the 500 to Configurations.Time_To_Unload

File: src/main/Elevator.java [hunk 2]
```diff
@@ -216,9 +227,10 @@ public void run() {
 		            tempElevator.setDoorsOpen(false);
 		        }
 		    };
+		    
 		    Timer timer = new Timer("Timer");
-		    long delay = fe == null ? 500 : 
-		    	fe.getErrorCode() == 1 ? 1000 : 500;
+		    long delay = fe == null ? Configurations.TIME_TO_LOAD_UNLOAD: 
+		    	fe.getErrorCode() == 1 ? 1000 : Configurations.TIME_TO_LOAD_UNLOAD;
```
Comment (AashnaNarang) [inline src/main/Elevator.java:233]: @krishane2018 can you double check this is correct and should we make change the 1000 into ```Configurations.TIME_TO_LOAD_UNLOAD * 2```?

Comment (yuvrajzala26) [reply src/main/Elevator.java:233]: fixed it.

**Pull request closed without merging**